### DISCUSSION TITLE
Key Recovery when verifier is deleted

### DISCUSSION THREAD

**Discussion by ankushagarwal** [Web3Auth/Web3Auth#1257] [Help]
Let's say I have a verifier named `ankush-custom-verifier` and I accidentally delete it, do I lose all the keys associated with the verifier?

Is there any way to recover it? 

Is it possible to recreate the same verifier with the same params (and recover the keys)?

**Comment by girishvi** [comment]
@ankushagarwal You would need to contact our team in order to recover the keys and verifier

**Comment by ankushagarwal** [comment]
Got it, thanks @girishvi!

**Comment by refi93** [comment]
I guess the same applies to "plug & play" projects, right? @girishvi
